buildSrc/src/main/kotlin/otel.java-conventions.gradle.kts

```diff
@@ -113,6 +113,14 @@ tasks {
       )
     }
 
+    val defaultMaxRetries = if (System.getenv().containsKey("CI")) 2 else 0
```

Comment:
opentelemetry-java-instrumentation retries 5 times. Want to see if we can get away with a more conservative figure.